topics/yamlconfig.rst

```diff
@@ -205,6 +205,73 @@ documentation <doxygen>`.
   describe the component. However, there is support for reflection by
   mnemonic name.
 
+  For example, you could have a stellar disk component and a dark matter halo component. 
+  The **dark halo** is the dark matter halo with a spherical basis, where parameters can 
+  be set as follows:
+
+.. code-block:: python
+
+   # Make the halo basis config
+  
+   Components:
+      - name:      dark halo             #component name, this is up to you
+        bodyfile:  halo.bods             #body file, this can be omitted if you are only interested in making a basis and not a simulation
+        parameters:
+           #leaving blank to set to defaults, these are expert parameters
+        force
+            id:    sphereSL              #adaptive basis, here the sphereSL type
+        parameters :
+            numr:  2000                  # number of radial grid points
+            rmin:  0.0001                # smallest radial grid point
+            rmax:  1.95                  # largest radial grid point
+            Lmax:  4                     # maximum spherical harmonic degree
+            nmax:  10                    # maximum radial order
+            rmapping: 0.0667             # radius for coordinate mapping
+            modelname: SLGridSph.model   # model file name
+            Cachename: slgrid_sph_cache  # cache name, this can be omitted if you are only interested in making a basis and not a simulation 
+
+
+The parameters *rmin* and *rmax* define the inner and outer radii for the basis in virial units
```

Comment:
I like to use *virial* units, but `exp` has no requirement on the units except that ``G=1``.